src/main/groovy/org/edx/jenkins/dsl/AnalyticsConstants.groovy

```diff
@@ -0,0 +1,104 @@
+package org.edx.jenkins.dsl
+
+class AnalyticsConstants {
+
+    public static def common_log_rotator = { extraVars ->
+        return {
+            daysToKeep(30)
+        }
+    }
+
+    public static def common_multiscm = { extraVars ->
+        return {
+            git {
+                remote {
+                    url('$TASKS_REPO')
+                    branch('$TASKS_BRANCH')
+                }
+                extensions {
+                    relativeTargetDirectory('analytics-tasks')
+                    pruneBranches()
+                    cleanAfterCheckout()
+                }
+            }
+            git {
+                remote {
+                    url('$CONFIG_REPO')
+                    branch('$CONFIG_BRANCH')
+                }
+                extensions {
+                    relativeTargetDirectory('analytics-configuration')
+                    pruneBranches()
+                    cleanAfterCheckout()
+                }
+            }
+        }
+    }
+
+    public static def common_parameters = { extraVars, env=[:] ->
+        return {
+            stringParam('CLUSTER_NAME', env.get('CLUSTER_NAME', extraVars.get('CLUSTER_NAME')), 'Name of the EMR cluster to use for this job.')
+            stringParam('CONFIG_BRANCH', '$ANALYTICS_CONFIGURATION_RELEASE', 'e.g. tagname or origin/branchname, or $ANALYTICS_CONFIGURATION_RELEASE')
+            stringParam('CONFIG_REPO', 'git@github.com:edx/edx-analytics-configuration.git', '')
+            stringParam('EMR_MASTER_INSTANCE_TYPE', extraVars.get('EMR_MASTER_INSTANCE_TYPE', 'm4.2xlarge'), 'EC2 Instance type used for master.')
```

Comment:
Shouldn't this and other instance types vary by env as well as extraVars?  Or is the only way to scale differently for edge versus prod is to change the number of instances?